.github/workflows/sync_default_branch.yml

```diff
@@ -0,0 +1,27 @@
+name: Sync Default Branch with Target Branch
+
+on:
+  workflow_call:
+    inputs:
+      target_branch:
+        required: false
+        type: string
+        default: storybook-site
```

Comment:
even though we do use this github action for `storybook-site` often, having this as default doesn't make as much sense as the other defaults we have ('main' branch and generic script command like `npm run test`). I think we can either make this required or maybe have `develop` to be default instead?

Reply:
updated to make it required